### DISCUSSION TITLE
GoPRO12  - Cant update as the battery says 1% charged.

### DISCUSSION THREAD

**Discussion by siedlik** [gopro/labs#1078] [Q&A]
I'm trying to use the GoPro with a power bank, and it says low power. 
So now I'm trying to update it to GoPro Labs, but the battery from GoPro 10 says 1%, and it doesn't charge. The battery is original.
When I try to update it with just the cable plugged in, I get black screen only.
The battery is fully charged, I checked the voltage.

**Comment by dnewman-gpsw** [comment]
There is lot going wrong here.  For the update you require a charged camera original battery, no update can be performed with a charged battery. I expect a Blue HERO10 battery in HERO12 would not be sufficient, use the HERO12 white battery.

**Reply by siedlik** [reply]
Just bought another GoPro 12 and the battery is working fine. So I think the first gopro12 got water damaged or something. It's working fine, but it always shows battery level 1% with the battery in, so I can't install Labs Pro as it's reporting under 10% battery.
I'm trying to instal Labs Pro and skip the USB power check, as I'm trying to use the GoPro on motorcycle with USB power only.

**Reply by dnewman-gpsw** [reply]
Unfortunately you can't bypass the power check without Labs installed, you can't install Labs without battery reading good amount. Catch-22.  I don't know of a solution other than to replace the water damaged camera.
